schemas/attributes/unique.go

```diff
@@ -0,0 +1,34 @@
+package attributes
```

Comment:
I propose to move this package away from `schemas`.
The function above depends on an higher-level API than other `schemas`'s APIs (the need of `github.com/fabbricadigitale/scimd/api/attr` demonstrates that).

May be a package just for a single function it's not needed, and we could simple add this function to an already existing package.

Please, evaluate it.